.github/workflows/publish.yml

```
@@ -101,6 +101,17 @@ jobs:
             -H "Authorization: Bearer ${{ secrets.REMOTE_DISPATCH_TOKEN }}" \
             -d '{"ref":"develop"}' \
             https://api.github.com/repos/simple-icons/simple-icons-font/actions/workflows/auto-release.yml/dispatches
+  pdf:
+    name: Trigger simple-icons-pdf release
+    needs: npm
```

It probably makes more sense to make this depend on `sanity-check`, this is not technically dependent on the `npm` release.

```suggestion
    needs: sanity-check
```

(same for the `font` job)